spec/models/project_spec.rb

```diff
@@ -7,4 +7,45 @@
     it {should have_many :contestant_projects}
     it {should have_many(:contestants).through(:contestant_projects)}
   end
+
+  describe "instance variables" do
+    describe "#num_of_contestants" do
+      before do
+        @recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+        @furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+
+        @news_chic = @recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+        @boardfit = @recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+
+        @upholstery_tux = @furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+        @lit_fit = @furniture_challenge.projects.create(name: "Litfit", material: "Lamp")
+
+        @jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+        @gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+        @kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+        @erin = Contestant.create(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+
+        ContestantProject.create(contestant_id: @jay.id, project_id: @news_chic.id)
+        ContestantProject.create(contestant_id: @gretchen.id, project_id: @news_chic.id)
+        ContestantProject.create(contestant_id: @gretchen.id, project_id: @upholstery_tux.id)
+        ContestantProject.create(contestant_id: @kentaro.id, project_id: @upholstery_tux.id)
+        ContestantProject.create(contestant_id: @kentaro.id, project_id: @boardfit.id)
+      end
+
+      it "returns the count of the number of contestants on this project" do
+        
+        expect(@news_chic.num_of_contestants).to eq(2)
+        expect(@boardfit.num_of_contestants).to eq(1)
+        expect(@upholstery_tux.num_of_contestants).to eq(2)
+        expect(@lit_fit.num_of_contestants).to eq(0)
+      end
+
+      it "returns the average of years of experience of contestants on this project" do
+        
+        expect(@news_chic.avg_experience_of_contestants).to eq(12.5)
+        expect(@boardfit.avg_experience_of_contestants).to eq(8)
+        expect(@upholstery_tux.avg_experience_of_contestants).to eq(10)
+      end
+    end
```

Comment:
Nice work testing this functionality at the unit level (in a model test) and integration level (in a feature test)!!